jssource/src_files/include/javascript/EmailsComposeViewModal.js

```diff
@@ -114,10 +114,10 @@
     self.emailComposeView = null;
```

Comment:
This comment not related to the original issue about the incorrect JSON contents in responses.